lib/Ix/DBIC/ResultSet.pm

```diff
@@ -158,7 +158,7 @@ sub ix_get_updates ($self, $ctx, $arg = {}) {
 
   if ($statecmp->is_resync) {
     $ctx->error(cannotCalculateChanges => {
-      description => "client cache must be reconstucted"
+      description => "client cache must be reconstructed"
```

Comment:
This is where I saw the weird jcmp_deeply() failure using re('...'). Wasn't until later I realized it was because I had the correct "reconstructed" in the re() and the error actually didn't match...